projects/media-viewer/src/lib/annotations/annotation.component.html

```diff
@@ -1,21 +1,22 @@
-<div class="annotation-wrapper" tabindex="1" (focusout)="toggleSelection(false)">
```

Comment:
the tabindex and focusout event listener are needed to manage highlight/comment selection/deselection

Reply:
Will add it back.